deps/rabbitmq_peer_discovery_k8s/priv/schema/rabbitmq_peer_discovery_k8s.schema

```diff
@@ -76,10 +80,73 @@ fun(Conf) ->
     end
 end}.
 
+%% modern keys
+
+{mapping, "cluster_formation.k8s.tls.cacertfile", "rabbit.cluster_formation.peer_discovery_k8s.ssl_options.cacertfile",
+    [{datatype, string}, {validators, ["file_accessible"]}
+]}.
+
+{translation, "rabbit.cluster_formation.peer_discovery_k8s.ssl_options.cacertfile",
+fun(Conf) ->
+    case cuttlefish:conf_get("cluster_formation.k8s.tls.cacertfile", Conf, undefined) of
+        undefined -> cuttlefish:unset();
+        Value     -> Value
+    end
+end}.
+
+{mapping, "cluster_formation.k8s.tls.certfile", "rabbit.cluster_formation.peer_discovery_k8s.ssl_options.certfile",
```

Comment:
We use `tls` for prefix around definition import (from an HTTPS endpoint), so I figured it would make a bit more sense here compared to `ssl_options` if server listeners.